CHANGELOG.md

```diff
@@ -46,6 +46,15 @@ Main (unreleased)
 
 - Fix a race condition where the ui service was dependent on starting after the remotecfg service, which is not guaranteed. (@dehaansa & @erikbaranowski)
 
+- Update mysqld_exporter from v0.15.0 to v0.16.0, most notable changes: (@cristiangreco)
```

Comment:
Note that `v0.12.2-0.20231005125903-364b9c41e595` we were using before was actually pointing at a commit just after 0.15.0.

Reply:
Is the changelog format ok? I picked only the changes I thought are most relevant for the embedded exporter.

Reply:
the format is fine but it should not be in the ### Bugfixes area. I think that it can be in Enhancements or Other changes